SnapWarShareExtension/SWSharePageViewManager.swift

```diff
@@ -0,0 +1,53 @@
+class SWSharePageViewManager {
```

Comment:
Is this an example of a delegator you told me to read about C:

Reply:
nope! this is the manager i use to handle knowing what view controllers to render when swiping through the carousel. i'll point you out to where the delegate and datasource stuff is done